invenio_app_rdm/theme/webpack.py

```diff
@@ -12,42 +12,37 @@
 from invenio_assets.webpack import WebpackThemeBundle
 
 
-def theme():
-    """Returns module's webpack bundle.
-
-    This is a callable function in order to lazy load `current_app`
-    and avoid working outside application context.
-    """
-    return WebpackThemeBundle(
-        __name__,
-        'assets',
-        default='semantic-ui',
-        themes={
-            'semantic-ui': dict(
-                entry={
-                    'invenio-app-rdm-theme':
-                        current_app.config['INSTANCE_THEME_FILE'],
-                    'invenio-app-rdm-deposits-deposit-searchbar-js':
-                        './js/invenio_app_rdm/deposits/RDMDepositSearchBar/index.js',  # noqa
-                    'invenio-app-rdm-deposits-deposit-form-js':
-                        './js/invenio_app_rdm/deposits/RDMDepositForm/index.js',  # noqa
-                    'invenio-app-rdm-search-app-js':
-                        './js/invenio_app_rdm/search_app_customizations.js',
-                },
-                dependencies={
-                    # add any additional npm dependencies here...
-                    "@babel/runtime": "^7.9.0",
-                    'formik': '^2.1.4',
-                    'luxon': '^1.23.0',
-                    'path': '^0.12.7',
-                    'prop-types': '^15.7.2',
-                    'react-invenio-deposit': '^0.2.0',
-                    'react-invenio-forms': '^0.3.0',
-                },
-                aliases={
-                    '../../theme.config$': 'less/theme.config',
-                    '@templates': "templates",
-                }
-            ),
-        }
-    )
+theme = WebpackThemeBundle(
+    __name__,
+    'assets',
+    default='semantic-ui',
+    themes={
+        'semantic-ui': dict(
+            entry={
+                'invenio-app-rdm-deposits-deposit-searchbar-js':
```

Comment:
@lnielsen I think this can be deleted too, as now we use the main `header.html`. You can delete all occurrences of this bundle + plus the code.